### DISCUSSION TITLE
No OCR when PDF to Word ?

### DISCUSSION THREAD

**Discussion by Chico0008** [Stirling-Tools/Stirling-PDF#1271] [Q&A]
Hi
I may be missing some things but
when a do a PDF to Word convertion, i was expecting Stirling to OCD my Pdf and change it to a doc/odt file.
but the result is just Image from pdf to a doc file.

can't edit the text in my doc, so am i missing something ?

**Comment by Frooodle** [comment]
We don't run OCR on conversion on convert 
We convert the PDF itself unchanged

**Reply by Chico0008** [reply]
what do you mean convert the PDF itself ?

So How can i get a PDF to Doc File, with the possibilty to edit text ?
my Pdf contain text and some small image/logo in corner

Adobe can do that but only on Microsoft Format, not opendoc
